compose/buildlocal.sh

```diff
@@ -9,15 +9,18 @@ GALAXY_REPO=galaxyproject/galaxy
 
 DOCKER_ADDITIONAL_BUILD_ARGS="--no-cache"
 
+# For using latest simply leave this variable empty or set to ":latest". This should be the case on the master branch.
+TAG=":dev"
```

Comment:
The Dockerfile changes looks good to me.
The buildlocal.sh changes will probably not work as travis is using this script, but than starts the `:latest` container (without specifying them).
So maybe setting this line to `:latest` by default will make at least travis happy.
For development people can change this easily.

Reply:
Sure!